assets/custom-icons/_data/custom-icons.json

```diff
@@ -108,6 +108,9 @@
       "slug": "Gosuslugi",
       "hex": "EE2F53"
     },
+    {
+      "title": "Healthchecks.io"
```

Comment:
Could you add the slug to this as well? Since the file name is simply `healthchecks`

Reply:
Done :D